### DISCUSSION TITLE
Charts' name/key disapear in the new version

### DISCUSSION THREAD

**Discussion by SoraKasvgano** [netdata/netdata#16415] [Q&A] [answered]
Charts' name/key disappear in the new version

Hi netdata friends
   recently, i am deploying netdata supervisor screen.....and using the newest official docker image to deploy netdata.
but i found  i can't get the chart's Name/key from the web UI, i have tried V0/V1/V2,but all of them can't be disaplayed....
is the configuration error?or it just gone in the new version?

for example,name/key is in the image below at right side.
<img width="1267" alt="temperature-config" src="https://github.com/netdata/netdata/assets/8521327/1ad2cb07-bd55-495b-ab6c-7b4b60036064">

Thanks....

<img width="240" alt="no-temperature" src="https://github.com/netdata/netdata/assets/8521327/60557480-0fa3-46f9-a417-d8f8dbb192ac">
https://gitee.com/dannylsl/routermonitor

**Comment by SoraKasvgano** [comment]
ohhh looks like we can search here
:19999/api/v1/charts
`sensors.temp_iio_hwmon_hwmon_hwmon0_iio_hwmon`
`sensors.temp_thermal_zone0_thermal_thermal_zone0_thermal_zone0`
but how to enable it in webui?

**Comment by ilyam8** [comment, marked as answer]
Fixed in https://github.com/netdata/netdata/pull/16416

**Reply by SoraKasvgano** [reply]
Thanks dude! it  would be more friendly for those who wants to develop new toys!
